### DISCUSSION TITLE
API's version

### DISCUSSION THREAD

**Discussion by praneshvelu** [community/community#140844] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all, I'm working on a research. Where i need some help in finding what are all the API's GitHub is having and what are the deprecated versions till date. Can some one suggest a way to find these details.

**Comment by buraito-kun** [comment]
Hello @praneshvelu. I think this will help you: https://docs.github.com/en/rest/quickstart?apiVersion=2022-11-28. It seems like the API version 2022-11-28 is the latest version.

From the sidebar, you can see what API's GitHub provides.
